explainer/openai_api.py

```diff
@@ -0,0 +1,22 @@
+import os
+import openai
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/